stories/features/sync/disabledContent.tsx

```diff
@@ -0,0 +1,78 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+
+// Components
+import Button from '../../../src/components/buttonsIndicators/button'
+
+// Component-specific components
+import { Grid } from '../../../src/features/sync'
+
+// Modals
+import NewToSyncModal from './modals/newToSync'
+import ExistingSyncCodeModal from './modals/existingSyncCode'
+
+// Utils
+import locale from './page/fakeLocale'
+
+interface SyncDisabledContentState {
+  newToSyncModal: boolean
```

Comment:
nit: using the word 'modal' in this prop ties the implementation to the caller, who does not need to know that a modal will be used. Presumably the caller can just say that this is either 'newToSync' or 'existintSyncCode'? Further, could it just be one boolean - is it possible for both to be true or both to be false?

Reply:
it is possible for both to be `false` only, which is the initial state. Removed the "modal" word